apps/src/maze/beeBlocks.js

```diff
@@ -201,6 +226,44 @@ function addIfElseFlowerHive(blockly, generator) {
   };
 }
 
+function addRepeatedActionBlock(
```

Comment:
can we unit test this?

Reply:
I don't think we have a good way of unit testing block creation. I looked at existing code/tests and couldn't find any similar code that has test coverage.
I spent some time trying to set up a test case but I would have to mock basically everything about blockly, at which point I don't think the test would have much value- it would basically just assert that the right functions are called in the right order, nothing about the actual behavior.
Since there aren't any tests for any of the existing block definitions, I feel sort of okay with not having a test here, but what do you think?

Reply:
> I would have to mock basically everything about blockly

that's the part i was wondering about. if that's the case, then a unit test here won't tell us much, so i think we can leave as-is